fc/parsing/CompactSyntaxParser.py

```diff
@@ -1025,102 +1027,102 @@ class CompactSyntaxParser(object):
-    lambdaExpr = p.Group(MakeKw('lambda') - paramList + ((colon - expr) | (obrace - stmtList + embedded_cbrace))
+    param_decl = p.Group(nc_ident_as_var + Optional(eq + expr))
+    param_list = p.Group(optional_delimited_list(param_decl, comma))
+    lambda_expr = p.Group(make_kw('lambda') - param_list + ((colon - expr) | (obrace - stmt_list + embedded_cbrace))
                          ).setName('Lambda').setParseAction(Actions.Lambda)
 
     # Function calls
     # TODO: Allow lambdas, not just ident?
-    argList = p.Group(OptionalDelimitedList(expr, comma))
-    functionCall = p.Group(identAsVar + Adjacent(oparen) - argList +
+    arg_list = p.Group(optional_delimited_list(expr, comma))
+    function_call = p.Group(ident_as_var + adjacent(oparen) - arg_list +
                            cparen).setName('FnCall').setParseAction(Actions.FunctionCall)
 
     # Tuples
-    tuple = p.Group(oparen + expr + comma - OptionalDelimitedList(expr, comma) +
+    tuple = p.Group(oparen + expr + comma - optional_delimited_list(expr, comma) +
                     cparen).setName('Tuple').setParseAction(Actions.Tuple)
 
     # Accessors
-    accessor = p.Combine(Adjacent(p.Suppress('.')) -
+    accessor = p.Combine(adjacent(p.Suppress('.')) -
                          p.oneOf('IS_SIMPLE_VALUE IS_ARRAY IS_STRING IS_TUPLE IS_FUNCTION IS_NULL IS_DEFAULT '
                                  'NUM_DIMS NUM_ELEMENTS SHAPE')).setName('Accessor')
 
     # Indexing
-    pad = (MakeKw('pad') + Adjacent(colon) - expr + eq + expr).setResultsName('pad')
-    shrink = (MakeKw('shrink') + Adjacent(colon) - expr).setResultsName('shrink')
+    pad = (make_kw('pad') + adjacent(colon) - expr + eq + expr).setResultsName('pad')
+    shrink = (make_kw('shrink') + adjacent(colon) - expr).setResultsName('shrink')
     index_dim = expr.setResultsName('dim')
-    index = p.Group(Adjacent(p.Suppress('{')) - expr +
+    index = p.Group(adjacent(p.Suppress('{')) - expr +
                     p.ZeroOrMore(comma - (pad | shrink | index_dim)) + p.Suppress('}')).setName('Index')
 
     # Special values
-    nullValue = p.Group(MakeKw('null')).setName('Null').setParseAction(Actions.Symbol('null'))
-    defaultValue = p.Group(MakeKw('default')).setName('Default').setParseAction(Actions.Symbol('defaultParameter'))
-    stringValue = quotedString.copy().setName('String').setParseAction(Actions.Symbol('string'))
+    null_value = p.Group(make_kw('null')).setName('Null').setParseAction(Actions.Symbol('null'))
+    default_value = p.Group(make_kw('default')).setName('Default').setParseAction(Actions.Symbol('defaultParameter'))
+    string_value = quoted_string.copy().setName('String').setParseAction(Actions.Symbol('string'))
 
     # Recognised MathML operators
-    mathmlOperators = set('''quotient rem max min root xor abs floor ceiling exp ln log
+    mathml_operators = set('''quotient rem max min root xor abs floor ceiling exp ln log
```

Comment:
Tidy up the indentation on the next few lines?

Reply:
will try!